src/styles/app.styl

```diff
@@ -99,13 +99,18 @@ header
         box-shadow      none
 
 #maif-figure, #fing-figure
-    .svg-wrapper
-        background-color $maif
-
     svg
         max-width       100%
```

Comment:
@GoOz Do we still need those two width and height properties ? Aren't they redundant with 
```stylus
header
  figure
     svg
       max-width       (72 / 16)em
       height          (72 / 16)em
```